packages/gatsby-plugin-netlify/README.md

```diff
@@ -34,6 +34,7 @@ plugins: [
       mergeCachingHeaders: true, // boolean to turn off the default caching headers
       transformHeaders: (headers, path) => headers, // optional transform for manipulating headers under each path (e.g.sorting), etc.
       generateMatchPathRewrites: true, // boolean to turn off automatic creation of redirect rules for client only paths
+      contentHashes: true, // optional boolean to remove content hashes for assets generated
```

Comment:
Mhhh. Now I read it again. But I think it is a little bit confusing now that readme part with the `true` option after my suggested change

- Now the comment not match the function of this option as true

- Maybe should some more or different notes here - (default is true but for netlify suggested to change to false)

I am a little unsure here. 

I think the default is true so that your PR is not a breaking change?

cc @marcysutton pls review for better wording

Reply:
Correct the default is `true` so it is not a breaking change. The explanation further down should explain the appeal of setting this to `false` for Netlify users.